### PR TITLE
Make adjustments to site layout and content

### PR DESCRIPTION
**Pull request by eirizarry**
These changes were made to finalize the way the content looks on the refurbished WorldWind site. They include the following: 

- Spelling and grammatical corrections to site content
- Making the appearances of the WorldWind projects consistent throughout the site
- Adding/fixing links to the appropriate site pages or resources
- Adjusting content layout to work well with responsiveness 

Closes [#58](https://github.com/NASAWorldWind/NASAWorldWind.github.io/issues/58)

### CODE REVIEW

File: content/android/tutorials/_index.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ---
-title: "Tutorials"
+title: "WorldWind Android Tutorials"
 date: 2017-06-30T14:51:58-04:00
 draft: false
 uselisttemplate: true
 ---
 
 ## Tutorials
```
Comment (pdavidc) [inline content/android/tutorials/_index.md:9]: There's some odd spacing in the Android tutorials opening paragraph. It doesn't show up in the final product, but ought to be addressed anyway.

File: content/java/tutorials/build-a-custom-renderable.md [hunk 2]
```diff
@@ -237,62 +236,62 @@ Here are the changes required to support ordered rendering:
 ```java
 protected class OrderedCube implements OrderedRenderable
 {
```
Comment (pdavidc) [inline content/java/tutorials/build-a-custom-renderable.md:238]: Spaces erroneously replaced by question marks.

File: content/java/tutorials/build-a-custom-renderable.md [hunk 3]
```diff
@@ -357,27 +356,27 @@ Finally, we'll add checks to avoid drawing the cube if it is not visible, or if
 ```java
 public void render(DrawContext dc)
 {
```
Comment (pdavidc) [inline content/java/tutorials/build-a-custom-renderable.md:358]: Spaces erroneously replaced by question marks.

File: content/java/get-started.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
 ---
-title: "Get Started"
+title: "WorldWind Java Get Started"
 date: 2017-07-27T09:50:17-05:00
 draft: false
 ---
```
Comment (pdavidc) [inline content/java/get-started.md:6]: The numbered lists don't render in the correct order, and their formatting isn't as good a fit for the new Hugo site. See https://goworldwind.org/getting-started/ for the original intended formatting.

**Review by pdavidc — CHANGES_REQUESTED** [review]
Very nice final result, with a few changes needed before we can merge.

**Comment by eirizarry** [comment]
The last few formatting issues have been corrected.

**Comment by pdavidc** [comment]
Looks great.

**Review by pdavidc — APPROVED** [review]

**Pull request merged**